extenders/Message.js

```diff
@@ -1,18 +1,29 @@
 const { Structures } = require("discord.js");
 
 module.exports = Structures.extend("Message", DiscordMessage => {
-  return class Message extends DiscordMessage {
+  return class MisakiMessage extends DiscordMessage {
```

Comment:
please dont do this not every message object needs to be renamed

Reply:
Makes Misaki look cool :p